assets/model_monitoring/components/src/model_data_collector_preprocessor/mdc_preprocessor_helper.py

```diff
@@ -166,6 +179,99 @@ def cross_hour(start, end) -> List[str]:
     return cross_year(start_datetime, end_datetime)
 
 
+def copy_appendblob_to_blockblob(appendblob_url: StoreUrl, start_datetime: datetime, end_datetime: datetime):
+    """Copy append blob to block blob."""
+    datastore_type = None if appendblob_url._datastore is None else appendblob_url._datastore.datastore_type
+    sas_token = _get_sas_token(appendblob_url.account_name, appendblob_url.container_name,
+                               appendblob_url.get_credential(), datastore_type)
+    base_path = appendblob_url.path
+
+    if datastore_type == "AzureBlob":
+        return _blob_copy_appendblob_to_blockblob(appendblob_url.get_container_client, base_path,
+                                                  start_datetime, end_datetime, sas_token)
+    elif datastore_type == "AzureDataLakeStorageGen2":
+        return _gen2_copy_appendblob_to_blockblob(appendblob_url.get_container_client, base_path,
```

Comment:
Please check do we need that?

Reply:
checked, we don't need this workaround for gen2, removed gen2 related code